### PR TITLE
BaSP-MR-267: adjust landing style

### PR DESCRIPTION
**Pull request by ghost**
Define aesthetic changes in the landing page:
- Change the main front page to the link commented below.
- Fix animations.
- Add frames in the trainer view.
- Form: use shared components.
- Footer: add icons in the contact section and changes social media icons, replace all for mui icons.
- Footer: change background color .

Trello: https://trello.com/c/BGAqtXOo/267-css-adjust-landing-styles

### CODE REVIEW

**Comment by ghost** [comment]
[[CSS] - Adjust landing styles](https://trello.com/c/BGAqtXOo/267-css-adjust-landing-styles)

File: src/Components/LandingPage/landing.module.css [hunk 1]
```diff
@@ -35,18 +33,61 @@
...
   text-align: center;
   font-size: 1.3rem;
   letter-spacing: 0.1rem;
-  cursor: default;
   text-transform: capitalize;
+  cursor: default;
 }
 
-.bullet {
+/* .bullet {
```
Comment (AleMac17) [inline src/Components/LandingPage/landing.module.css]: Please, can you remove the comment lines

File: src/Components/LandingPage/landing.module.css [hunk 2]
```diff
@@ -1,30 +1,28 @@
...
-  object-position: center;
-  opacity: 0.8;
-  filter:
-  brightness(70%)
-  saturate(100%)
-  blur(2px)
-  sepia(10%)
-  contrast(100%)
-  grayscale(100%);
+  /* object-position: top; */
```
Comment (AleMac17) [inline src/Components/LandingPage/landing.module.css]: Please, can you remove the comment lines

File: src/Components/LandingPage/join.module.css [hunk 3]
```diff
@@ -59,35 +58,33 @@
...
 
-.landscape Button:hover {
-  font-size: 7rem!important;
+/* .landscape Button:hover {
   text-shadow: 10px 10px 10px rgb(0, 0, 0);
-  color: white!important;
-  transition: color 0.5s ease, font-size 1s ease, text-shadow 1s ease;
-}
\ No newline at end of file
+  transition: 1s ease;
```
Comment (AleMac17) [inline src/Components/LandingPage/join.module.css]: Please, can you remove the comment lines

File: src/Components/LandingPage/join.module.css [hunk 4]
```diff
@@ -59,35 +58,33 @@
...
   text-transform: uppercase;
-  color: rgba(0, 0, 0, 0)!important;
-  transition: color 1s ease, font-size 1s ease, text-shadow 1s ease;
+  font-size: 10rem;
+  color: white;
 }
 
-.landscape Button:hover {
-  font-size: 7rem!important;
+/* .landscape Button:hover {
```
Comment (AleMac17) [inline src/Components/LandingPage/join.module.css]: Please, can you remove the comment lines

File: public/assets/images/landing/landscape.jpg
Comment (AleMac17) [inline public/assets/images/landing/landscape.jpg:1]: Please, can you use tinyjpg to reduce the size?

File: public/assets/images/landing/portrait.jpg
Comment (AleMac17) [inline public/assets/images/landing/portrait.jpg:1]: Please, can you use tinyjpg to reduce the size?

**Review by AleMac17 — APPROVED** [review]

**Review by Gonzarey96 — APPROVED** [review]

**Pull request merged**